app/static/src/app/components/options/NumericInput.vue

```diff
@@ -37,11 +43,11 @@ export default defineComponent({
             default: ""
         },
         maxAllowed: {
-            type: Number,
+            type: [Number, Array] as PropType<number | BoundTooltip[]>,
             default: Infinity
         },
         minAllowed: {
-            type: Number,
+            type: [Number, Array] as PropType<number | BoundTooltip[]>,
```

Comment:
`maxAllowed` and `minAllowed` might be the wrong term now that some variants are warnings/info

Reply:
hmm yes but i do think `maxAllowed="0"` in a vue prop reads very nicely, i think they are doing max validation even with the warning variant just not fully limiting it so i do get why you want to change it because we do allow stuff past one of the maxes. maybe `maxThresholds`? or something like that? `maxBounds` perhaps? im not very good with names